apps/application-system/api/src/app/modules/payment/payment.service.ts

```diff
@@ -13,6 +13,8 @@ import { CreateChargeResult } from './payment.type'
 import { logger } from '@island.is/logging'
 import { ApolloError } from 'apollo-server-express'
 import { Application as ApplicationModel } from '@island.is/application/api/core'
+import { unknown } from 'zod'
```

Comment:
use a different name as unknownZod or something to differentiate from unknown type